cookbooks/travis_ci_ruby/attributes/default.rb

```diff
@@ -1,11 +1,11 @@
-override['rvm']['default'] = '2.2.5'
+override['rvm']['default'] = '2.2.6'
 override['rvm']['rubies'] = %w(
   1.9.2
   1.9.3
   2.0.0
   2.1.10
-  2.2.5
-  2.3.1
```

Comment:
Hey, @adelcambre,

Thanks for sharing your concerns. Having all (or several) Ruby versions is out of the question, I am afraid. We would want to standardize language runtimes policy, and reducing the number of runtimes, regardless of the language, is important.

Thank you for your understanding.